src/components/ENSAddress.tsx

```diff
@@ -28,18 +28,35 @@ function ENSAddressSuspended({
   )
 }
 
-export default memo<ENSAddressProps>(({ address, network }) => {
-  const { md, lg } = useBreakpoints()
-  const truncateSize = md ? (lg ? 25 : 17) : 11
-
+function ENSAddress({
+  address,
+  network,
+  truncateSize,
+}: ENSAddressProps & { truncateSize: number }) {
   return (
     <Suspense fallback={truncateMiddleIfNeeded(address, truncateSize)}>
       <ENSAddressSuspended
+        truncate
         address={address}
-        truncateSize={truncateSize}
-        truncate={!lg}
         network={network}
+        truncateSize={truncateSize}
       />
     </Suspense>
   )
+}
+
+export default memo<ENSAddressProps>(({ address, network }) => {
+  return (
+    <span>
+      <span className={display('block', 'md:hidden')}>
+        <ENSAddress address={address} network={network} truncateSize={11} />
+      </span>
+      <span className={display('hidden', 'md:block', 'lg:hidden')}>
```

Comment:
You probably need `lg:!hidden`, because it may not override previous style

Reply:
Why don't you use `displayFromMd` + `lg:!hidden`?

Reply:
Changed